examples/10-visit/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/lovoo/goka"
+	"github.com/lovoo/goka/codec"
+	"github.com/lovoo/goka/multierr"
+)
+
+var (
+	brokers             = []string{"localhost:9092"}
+	topic   goka.Stream = "example-migration-clicks-input"
+	group   goka.Group  = "example-migration-group"
+
+	tmc *goka.TopicManagerConfig
+)
+
+func init() {
+	// This sets the default replication to 1. If you have more then one broker
+	// the default configuration can be used.
+	tmc = goka.NewTopicManagerConfig()
+	tmc.Table.Replication = 1
+	tmc.Stream.Replication = 1
+}
+
+// emit messages until stopped
+func runEmitter(ctx context.Context) {
+	emitter, err := goka.NewEmitter(brokers, topic, new(codec.Int64))
+	if err != nil {
+		log.Fatalf("error creating emitter: %v", err)
+	}
+	defer emitter.Finish()
+	ticker := time.NewTicker(100 * time.Millisecond)
+	defer ticker.Stop()
+	for i := 0; ; i++ {
+		select {
+		case <-ticker.C:
+
+			err = emitter.EmitSync(fmt.Sprintf("key-%d", i%10), int64(1))
+			if err != nil {
+				log.Fatalf("error emitting message: %v", err)
+			}
+		case <-ctx.Done():
+			return
+		}
+	}
+}
+
+func main() {
+
+	tm, err := goka.NewTopicManager(brokers, goka.DefaultConfig(), tmc)
+	if err != nil {
+		log.Fatalf("Error creating topic manager: %v", err)
+	}
+	err = tm.EnsureStreamExists(string(topic), 8)
```

Comment:
why it is 8? we have 20 partitions, right?

Reply:
that's a completely arbitrary number here, can be 8 or 20 or 100. It's just an example how to use the visitor-tool. This should work with any number of partitions